08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,49 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            number : 0
+        }
+    }
+
+    static propTypes = {
+        number: React.propTypes.number.isRequired
```

Comment:
No reason to use propTypes here, because you don't pass number as a prop from outside the component. But if you do want to pass it from outside as initial number consider setting it inside the constructor as initial number. Anyway. It should be `React.PropTypes` and not React.propTypes